readme.md

```diff
@@ -0,0 +1,52 @@
+# GMod-Seamless-Portals
+[![](https://img.youtube.com/vi/lgiPHZdTGxs/0.jpg)](http://www.youtube.com/watch?v=lgiPHZdTGxs "")
+
+### Description
+My seamless portals addon for garrys mod. This is a mod that adds a entity called the Seamless Portal, and a functioning portal gun!
+You can spawn the Portals in the spawnmenu, or by using the portal gun in the weapons tab.
+
+### Installation
+To install this addon, download and unzip the contents and put it in your garrys mod addon folder
+
+### Features
+ * More seamless
```

Comment:
Than what?

Reply:
> Why are you lowercasing README and LICENSE? Goes against a lot of tooling and just general guidelines. Also LICENSE can be changed to LICENSE.md.

If you unzip an addon it will not get recognized on servers running on certain linux systems